src/include/OpenImageIO/typedesc.h

```diff
@@ -295,6 +295,8 @@ struct OIIO_API TypeDesc {
     static const TypeDesc TypeVector;
     static const TypeDesc TypeNormal;
     static const TypeDesc TypeMatrix;
+    static const TypeDesc TypeMatrix33;
+    static const TypeDesc TypeMatrix44;
```

Comment:
I don't have strong feeling one way or the other. I can take it out and remove the redundancy. I agree that it doesn't add anything.

After the final switch to C++11, these will get changed from static const to constexpr anyway.